### PR TITLE
Fix failing FAT for com.ibm.ws.logging.hpel_fat

### LINKED ISSUE #11945: Test Failure (20200421-1807): junit.framework.TestSuite.com.ibm.ws.logging.hpel.fat.FATSuite
RTC 273904

```
java.lang.RuntimeException: java.lang.RuntimeException: Error getting server
at componenttest.custom.junit.runner.FATRunner.injectLibertyServers(FATRunner.java:721)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:166)
Caused by: java.lang.RuntimeException: Error getting server
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:204)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:97)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:67)
at componenttest.custom.junit.runner.FATRunner.injectLibertyServers(FATRunner.java:687)
Caused by: componenttest.exception.TopologyException: Unable to delete pre-existing server directory: C:/jazz-build-engines/jazz112.hursley.ibm.com/EBC.PROD.WASRTC/build/dev/image/output/wlp/usr/servers/HpelServer
at componenttest.topology.impl.LibertyServer.setup(LibertyServer.java:647)
at componenttest.topology.impl.LibertyServer.<init>(LibertyServer.java:588)
at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:160)
```

### PR DESCRIPTION
**Pull request by Yushan-Lin**
Remove check for binaryLogging enablement bootstrap.properties file since in Windows platforms it can't delete the file if it's being opened. binaryLogging is always enabled in the servers bootstrap.properties.
Fixes  #11945

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

**Comment by fmhwong** [comment]
@Yushan-Lin This code was ported from WS-CD.  Why we didn't have this problem before?

File: dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELHideMessagesTest.java [hunk 1]
```diff
@@ -79,8 +78,8 @@ public static void setUp() throws Exception {
         // Setting the server.xml with the hideMessages logging attribute
         server.updateServerConfiguration(new File(server.pathToAutoFVTTestFiles, "server-HPELHideMessagesTest.xml"));
         // Restart server
-        server.restartServer();
-
+        server.stopServer();
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELHideMessagesTest.java:81]: Not sure why we need this change as `server.restartServer()` should do `server.stopServer()` and `server.startServer()`.

File: dev/com.ibm.ws.logging.hpel_fat/publish/servers/HpelServer2/bootstrap.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
 bootstrap.include=../testports.properties
 websphere.log.provider=binaryLogging-1.0
 com.ibm.ws.logging.console.log.level=INFO
+com.ibm.ws.logging.trace.specification=com.ibm.ws.logging.*=all:com.ibm.ws.org.*=all=enabled
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.hpel_fat/publish/servers/HpelServer2/bootstrap.properties:4]: How much trace is generated at the end of the test?

Comment (Yushan-Lin) [reply dev/com.ibm.ws.logging.hpel_fat/publish/servers/HpelServer2/bootstrap.properties:4]: The binary tracedata is 64KB.

**Comment by Yushan-Lin** [comment]
@fmhwong there's some refactoring that was done from the WS-CD side with removing `HpelSetup` and removing whatever dependencies on WS-CD side (such as removing VerboseTestCase and changing ApplicationServer to Server). Not sure if there's something different with how the tests is handled but it only has this issue on Windows.

**Comment by Yushan-Lin** [comment]
#run-libby-bot

**Pull request merged**